samples/matter/light_bulb/src/app_task.h

```diff
@@ -37,11 +42,25 @@ class AppTask {
 	static void ActionInitiated(LightingManager::Action aAction);
 	static void ActionCompleted(LightingManager::Action aAction);
 	static void ButtonEventHandler(uint32_t buttonState, uint32_t hasChanged);
+	static void TimerEventHandler(k_timer *timer);
 	static void ThreadProvisioningHandler(const chip::DeviceLayer::ChipDeviceEvent *event, intptr_t arg);
+	static int SoftwareUpdateConfirmationHandler(uint32_t offset, uint32_t size, void *arg);
 
 	friend AppTask &GetAppTask();
 
+	enum Function_t {
```

Comment:
Would `TimerFunction` be more descriptive name? Also consider using `enum class` to get rid off the `kFunction_` prefix.